cmd/k8s-proxy/k8s-proxy.go

```diff
@@ -41,8 +42,9 @@ func main() {
 
 func run(logger *zap.SugaredLogger) error {
 	var (
-		configFile = os.Getenv("TS_K8S_PROXY_CONFIG")
-		podUID     = os.Getenv("POD_UID")
+		configFile  = os.Getenv("TS_K8S_PROXY_CONFIG")
+		loginServer = strings.TrimSuffix(os.Getenv("TS_K8S_PROXY_LOGIN_SERVER"), "/")
```

Comment:
Could we add it to the config file instead? I'd like to do as much as possible in there so there's a single consistent source of truth for config

Reply:
Sure, I've made that change, please take a look!